app/views/latest/index.html.erb

```diff
@@ -1,71 +1,79 @@
-<% page_title t("document.latest_from", subject: subject) %>
+<% page_title t("document.latest") %>
 <% page_class "documents-index" %>
 <% atom_discovery_link_tag filter_atom_feed_url, "Recent documents" %>
 
-<header class="block headings-block">
-  <div class="govuk-width-container">
-    <% if  subject.is_a? Organisation %>
-      <%= render "govuk_publishing_components/components/breadcrumbs", {
-        breadcrumbs: [
-          {
-            title: "Home",
-            url: "/",
-          },
-          {
-            title: "Organisations",
-            url: "/government/organisations",
-          },
-          {
-            title: "#{subject}",
-            url: "/government/organisations/#{subject['slug']}",
-          },
-        ]
-      } %>
-    <% elsif subject.is_a? TopicalEvent %>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <% if  subject.is_a? Organisation %>
         <%= render "govuk_publishing_components/components/breadcrumbs", {
```

Comment:
This could be tidier if we built the breadcrumbs array in the if and called the component to pass it to later - but it's beyond the scope of what's needed here.